components/ledger/internal/api/v2/controllers_accounts.go

```diff
@@ -80,55 +86,53 @@ func getAccount(w http.ResponseWriter, r *http.Request) {
 	}
 	pitFilter, err := getPITFilter(r)
 	if err != nil {
-		sharedapi.BadRequest(w, shared.ErrValidation, err)
+		sharedapi.BadRequest(w, ErrValidation, err)
 		return
 	}
 	query.PITFilter = *pitFilter
 
 	acc, err := l.GetAccountWithVolumes(r.Context(), query)
 	if err != nil {
-		shared.ResponseError(w, r, err)
+		sharedapi.InternalServerError(w, r, err)
```

Comment:
Same here s hould we send a not found error ?

Reply:
Same as above.